README.md

```diff
@@ -244,48 +159,35 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project share it with others
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank Microverse for providing reviewers who ensure that the best of this project was obtained
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
 
 ## ❓ FAQ <a name="faq"></a>
 
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
+- **How do I run SQL queries in the command line**
 
-  - [Answer_1]
+  - Run the usual SQL queries and ensure you terminate with "\g"
 
-- **[Question_2]**
+- **How do I access a table in a database in the command line**
 
-  - [Answer_2]
+  - Run the command "\d table_name"
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly fix the broken link to your `MIT` License agreement, for a more professional Readme file. This is where this link leads to:
![image](https://github.com/Adams-Ghub/vet-clinic-database/assets/81932478/50c4835c-bfbd-451d-9232-bfb1250c5132)